packages/tables/docs/10-advanced.md

```diff
@@ -256,6 +256,8 @@ public function table(Table $table): Table
         ]);
 ```
 
+You can pass `$table->icon()` and `$table->iconColor('primary')` method to add the icon in the header:
```

Comment:
Please add a separate section for this with a code snippet like the other APIs. Auto-generated screenshots can be added to schema.js and instructions to generate screenshots are in docs-assets/screenshots/script.js.

Reply:
I will do that